Clone: A very realistic clone of Apple home page

gabrie-lhilarion opened the pull request:
## IN THIS UPDATE I:
* Added the conspicuous navigation bar.
* Added the full background image.
* And used HTML semantic tags.

Commit:
created the following files

Commit:
added the following files
1. stickler.yml
2. stylelint.config.js

Commit:
Added some styles for the following
1. to centralize the navbar
2. to make the li elements to display horizontal and without a list style

Commit:
1.created styles to position the search icon and the search input properly on the navigation bar
2. added the reguired html

Commit:
added more html elements to build the page

Commit:
in this commit
1 the required images were added
2. the image containers were floated left
the footer element was added

Commit:
added a screenshot

gabrie-lhilarion committed:
Updated the readme file

gabrie-lhilarion committed:
Update README.md

Commit:
added the missing doctype declaration

Commit:
polished the css a bit by removing unneccesary spaces
'

Commit:
modified the read me file

Commit:
decreased the height of the page

Commit:
added padding to the search icon

thsvr commented:
### **STATUS: CHANGES REQUIRED** :muscle: :muscle: 

Hi @gabrie-lhilarion :+1: :+1: 

### **_You have been doing a GREAT job_** :1st_place_medal: :1st_place_medal: 
- Pull request are descriptive and professional
- Commits messages are clear and descriptive
- Linter is not showing any errors 
- Your README file is clear and descriptive
- You deployed your project on netlify :1st_place_medal: :1st_place_medal: :1st_place_medal: 
- Your clone looks pretty similar to the original one

**...and you are almost there** :bulb: 
There are just 3 simple required changes listed below, can you please take a look on them, do the changes and submit for another review?

:v: On the navbar we are supposed to use [background -> gradient](https://www.w3schools.com/cssref/css3_pr_background.asp). Can you please fix it? You can take a look on the link to have a clue about how to do it :smile_cat: 

:v: On the footer, the copyright should be a link also. Can you please fix it? 


If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)


### **Happy coding and good luck on the next review** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

Commit:
In this update-
1. I used background gradient instead of image for the navigation bar.
2. I made the Copyright text a link

gabrie-lhilarion commented:
## ALL  REQUIRED CHANGES AFFECTED
* The navbar has a background gradient
* The copyright text has been made a link

men32z reviewed (changes requested):
Hey @gabrie-lhilarion !
## Good making the changes! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

- [ ] there is a yellow background in your page that is not in the original page, remove it please.
- [ ] the articles( 4 images ) section should be the same size as the main image and be centered in wide screens. 

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

Commit:
updated the styleshet

Commit:
set the background-position value to cover in style.css

gabrie-lhilarion commented:
##RECOOMENDES CHANGE AFFECTED
* Background position set to cover

men32z reviewed (approved):
## Good job making the changes, you can merge.

![BlgS1sdIUAEybkW](https://user-images.githubusercontent.com/9115996/78264312-3df17a80-74c0-11ea-88a3-75d93f3a2d87.jpg)

### Status Approved

--Luis Preza
